s30020-NikiforKoseda/zad3.txt

```diff
@@ -0,0 +1,131 @@
+Ćwiczenia 1
+Napisz skrypt, który sprawdzi, czy podano więcej niż jeden parametr. Jeśli tak, to niech wyświetli pierwszy parametr. Jeśli nie, to niech wyświetli komunikat, że nie podano parametrów.
+
+###
+#!/bin/bash
+
+if [ "$#" -gt 1 ]; then
+    echo "$1"
+else
+    echo "This script does not have more than one parameter."
+fi
+
+Ten skrypt nauczył mnie ważnej rzeczy - SPACJE SĄ KONIECZNE PRZY NAWIASACH KWADRATOWYCH - inaczej skrypt nie działa. Oraz nauczyłem się -gt, nie można użyć < bo w linuxie to przekierowanie. Oraz $# przechowuje zmienne dla skryptu
```

Comment:
O spacjach mówiłem na wykładzie i pokazywałem także przykłady.
Znaków `<`, `<=`, `>=`, `>` można używać ale w rozszerzonym kontekście matematycznym dostępnym w podwójnych nawiasach. Natomiast są to bashyzmy, których należy unikach bo powodują trudnosci w portowaniu skryptów na inne powłoki np. `zsh`.

`$#` to ilość parametrów skryptu. Twoja wypowiedź jest nieprecyzyjna.